econml/cate_estimator.py

```diff
@@ -348,3 +408,94 @@ def coef__interval(self, *, alpha=0.1):
     @BaseCateEstimator._defer_to_inference
     def intercept__interval(self, *, alpha=0.1):
         pass
+
+
+class StatsModelsCateEstimatorDiscreteMixin(BaseCateEstimator):
```

Comment:
Consider refactoring to share some logic with `StatsModelsCateEstimatorMixin`

Reply:
I would again similarly push it to a future PR.

Reply:
Ok